### DISCUSSION TITLE
Dagster user management

### DISCUSSION THREAD

**Discussion by gianfrancodemarco** [dagster-io/dagster#23851] [Q&A] [answered]
Does Dagster have functionalities about users or it is only present in Dagster+? If not, how is it commonly used inside an organization?

**Comment by garethbrickman** [comment, marked as answer]
You are correct that these functionalities are only available in [Dagster+](https://dagster.io/plus) at this time. There is a Dagster Open Source feature request tracking support for user authentication and roles-based access control (RBAC) here: https://github.com/dagster-io/dagster/issues/2219
